trino/trino.go

```diff
@@ -690,15 +705,44 @@ func newErrQueryFailedFromResponse(resp *http.Response) *ErrQueryFailed {
 }
 
 type driverStmt struct {
-	conn           *Conn
-	query          string
-	user           string
-	nextURIs       chan string
-	httpResponses  chan *http.Response
-	queryResponses chan queryResponse
-	statsCh        chan QueryProgressInfo
-	errors         chan error
-	doneCh         chan struct{}
+	conn                          *Conn
+	query                         string
+	user                          string
+	nextURIs                      chan string
+	httpResponses                 chan *http.Response
+	queryResponses                chan queryResponse
+	statsCh                       chan QueryProgressInfo
+	usingSpooledProtocol          bool
+	spoolingMaxOutOfOrderSegments int
+	spoolingWorkerCount           int
+	spooledSegmentsMetadata       chan spooledMetadata
+	spooledSegmentToDecode        chan segmentToDecode
+	decodedSegment                chan decodedSegment
```

Comment:
Nit: use the plural form for all channels; it would be more consistent with existing channels and help distinguish channel names from struct names.
```suggestion
	decodedSegments                chan decodedSegment
```